src/components/HabitInviteList.tsx

```diff
@@ -0,0 +1,38 @@
+import { View } from "@/src/components/Themed";
+import HabitInvite from "@/src/components/HabitInvite";
+import { IconMusic, IconMoodTongue } from "@tabler/icons-react-native";
+import colors from "../constants/colors";
+
+// fetch all invites a user has received
+export default function HabitInviteList() {
+    const invites = [
+        { title: "Play Guitar", color: "purple", icon: IconMusic, numberOfParticipants: 3, userName: "blaze_kush" },
+        { title: "Yum Yum", color: "red", icon: IconMoodTongue, numberOfParticipants: 10, userName: "kush_blaze" }
+    ];
+
+    const deleteInvite = () => {
+        // delete the invite
+        console.log("Invite deleted");
+    };
+    const confirmInvite = () => {
+        // confirm the invite
+        console.log("Invite confirmed");
+    }
```

Comment:
Can you make confirm/deleting remove it from the list

Reply:
Or you could make this a new task on Linear

Reply:
i wanna add a popup on remove or confirm